docs/platform/webcontainers/troubleshooting.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Troubleshooting
+---
+
+# {{ $frontmatter.title }}
+
+This page helps you troubleshoot issues with WebContainers.
+
+## Cannot load native addon
+
+Currently, WebContainers can only execute languages that are natively supported on the Web  including JavaScript and WebAssembly. It is not possible to run [native addons](https://nodejs.org/api/addons.html) which are usually implemented using native languages such as C++, unless they can be compiled to WebAssembly. Therefore, loading native addons is disabled by default via [`--no-addons`](https://nodejs.org/api/cli.html#--no-addons) in WebContainers and you may encounter an error that says `Cannot load native addon because loading addons is disabled`. There is no way to opt-in into loading native addons because they can't be executed.
```

Comment:
```suggestion
Currently, WebContainers can only execute languages that are natively supported on the Web, including JavaScript and WebAssembly. It is not possible to run [native addons](https://nodejs.org/api/addons.html) which are usually implemented using native languages such as C++, unless they can be compiled to WebAssembly. Therefore, loading native addons is disabled by default via [`--no-addons`](https://nodejs.org/api/cli.html#--no-addons) in WebContainers. As a result, you may encounter an error that says: `Cannot load native addon because loading addons is disabled`. 

There is no way to opt-in into loading native addons because they can't be executed.
```

Reply:
I added two interpunction signs and a joining clause.